src/lib/actions/redirectWithIdentity/index.js

```diff
@@ -12,11 +12,13 @@ governing permissions and limitations under the License.
 
 const createRedirectWithIdentity = require("./createRedirectWithIdentity");
 const instanceManager = require("../../instanceManager/index");
+const createGetConfigOverrides = require("../../utils/createGetConfigOverrides");
 
 const document = window.document;
 
 module.exports = createRedirectWithIdentity({
   instanceManager,
   document,
-  logger: turbine.logger
+  logger: turbine.logger,
+  getConfigOverrides: createGetConfigOverrides(turbine.environment?.stage)
```

Comment:
Why are we passing the stage environment value?

Reply:
We need the stage (`development`, `production`, or `stage`) because overrides are now configure on a per-environment basis and we need to know which ones to get.